src/parser/csql_grammar.y

```diff
@@ -20765,8 +20799,36 @@ class_comment_spec
 			    PT_NODE_PRINT_VALUE_TO_TEXT (this_parser, node);
 			  }
 
+			PT_NODE *alter_node = parser_get_alter_node();
+
+			if (alter_node)
+			  {
+				alter_node->info.alter.code = PT_CHANGE_TABLE_COMMENT;
+				alter_node->info.alter.alter_clause.comment.tbl_comment = node;
+			  }
+
 			$$ = node;
```

Comment:
I think this can be removed.

Reply:
The return value is used in the syntax for changing the view comment. So I can't delete this code. The same file can be found at lines 3737→3749, 3760→3769.

Reply:
When using class_comment_spec, info.alter.code is not always PT_CHANGE_TABLE_COMMENT, so do not set the info.alter.code value to PT_CHANGE_TABLE_COMMENT in class_comment_spec. So, I changed it to PT_CHANGE_TABLE_COMMENT only when the info.alter.code value is not PT_CHANGE_COLUMN_COMMENT in the original place. So, i will change the info.alter.code value to be set to PT_CHANGE_TABLE_COMMENT only if it is not PT_CHANGE_COLUMN_COMMENT at the original location.